### PR TITLE
Solution for error No route matches [GET]

### PR DESCRIPTION
**Pull request by britaumich**
There is a TDX ticker submitted by Moni
[https://teamdynamix.umich.edu/TDNext/Apps/46/Tickets/TicketDet.aspx?TicketID=4775129](https://teamdynamix.umich.edu/TDNext/Apps/46/Tickets/TicketDet.aspx?TicketID=4775129)

She landed somehow on this URL: users/edit.

The pull request has a solution.
1. Removing registerable from the User model will give "No route matches [GET] "/users/edit"
2. To make this error look elegant I added a route and a method "not_found_method"

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
+  devise :database_authenticatable,
```
[the remark this answers is not in the document]
Comment (britaumich) [reply app/models/user.rb:24]: I removed recoverable here: [9ad4dc3](https://github.com/lsa-mis/mi_classrooms/pull/292/commits/9ad4dc39d5c9b60cc8a70701c8011a58ad5d68cb)

**Comment by britaumich** [comment]
@rsmoke I removed the non_found method

File: app/models/user.rb
Comment (rsmoke) [inline app/models/user.rb:1]: please also remove :validatable

**Review by rsmoke — CHANGES_REQUESTED** [review]
@britaumich, you can also delete the entire app/views/devise folder.

File: app/models/user.rb
Comment (britaumich) [reply app/models/user.rb:1]: done

**Comment by britaumich** [comment]
app/views/devise folder is removed

**Review by rsmoke — APPROVED** [review]

**Pull request merged**